src/common/util/status.h

```diff
@@ -235,6 +235,7 @@ class VINEYARD_MUST_USE_TYPE Status {
     }
   }
   Status(StatusCode, const std::string& msg);
+  Status(StatusCode, const std::string& msg, const std::string& bt);
```

Comment:
That is not need, as we shouldn't expect the user to specify the "backtrace".

Reply:
> That is not needed, as we shouldn't expect the user to specify the "backtrace".

Sure. This makes sense.